src/country-crisp-cereals.md

```diff
@@ -0,0 +1,37 @@
+# Country Crisp Cereals
+
+This is a recipe for making Jordan's Country Crisp-like cereals.
+
+- ⏲️ Prep time: 10 min
+- 🍳Cook time: 2 h
+- 🍽️ Servings: 7
+
+## Ingredients
+
+- Oat Flakes (270 g - 800 mL)
+- Rice Flour (or any other flour) (20 g - 55mL)
+- Sugar (100 g)
+- Crushed Hazelnuts (20 g) 
+- Oil (any type) (60 g - 75 mL)
+- Water (50 g)
+- Chocolate (70 g) (optional)
+- Vanilla Extract (1/2 ts) (optional)
+ 
+
```

Comment:
```suggestion
- Vanilla Extract (1/2 tsp) (optional)
 
```

Suggest removing the extra newline. Also, I assume `ts` is `tsp`?

Reply:
That's it, I wasn't know how to say this in english, thanks.